Mongle/Tuist/ProjectDescriptionHelpers/Project+Templates.swift

```diff
@@ -14,29 +14,48 @@ public extension Project {
         entitlements: Entitlements? = nil,
         schemes: [Scheme] = []
     ) -> Project {
-        
-        //TODO: Relase, Dev 버전에 맞춰 bundleSuffix를 변경해야함.
-        let bundleSuffix = "dev"
         let deploymentTarget = Environment.deploymentTarget
         var projectTargets: [Target] = []
+        
+        // MARK: TARGETS > Singing & Capabilities
+        let devSigningSettings: [String: SettingValue] = [
+            "PRODUCT_BUNDLE_IDENTIFIER": "com.mongle.dev",
+            "PROVISIONING_PROFILE_SPECIFIER": "Mongle.dev",
+            "CODE_SIGN_IDENTITY": "\(APIKey.codeSignIdentity)"
+        ]
+        let releaseSigningSettings: [String: SettingValue] = [
+            "PRODUCT_BUNDLE_IDENTIFIER": "com.mongle.release",
+            "PROVISIONING_PROFILE_SPECIFIER": "Mongle.release",
+            "CODE_SIGN_IDENTITY": "\(APIKey.codeSignIdentity)"
+        ]
         let settings: Settings? = .settings(
             base: [
-                "DEVELOPMENT_TEAM": "\(APIKey.developmentTeam)",
                 "CODE_SIGN_STYLE": "Manual",
-                "PROVISIONING_PROFILE_SPECIFIER": "Mongle.dev",
+                "DEVELOPMENT_TEAM": "\(APIKey.developmentTeam)",
                 "CODE_SIGN_IDENTITY": "\(APIKey.codeSignIdentity)"
             ],
             configurations: [
+                .debug(
+                    name: .configuration("Debug"),
+                    settings: devSigningSettings,
+                    xcconfig: .relativeToRoot("Config/Debug.xcconfig")
+                ),
+                .release(
+                    name: .configuration("Release"),
+                    settings: releaseSigningSettings,
+                    xcconfig: .relativeToRoot("Config/Release.xcconfig")
+                )
```

Comment:
위의 `signingSettings`함수를 사용한다면 아래처럼 함수를 호출하여 사용할 수 있습니다.
```suggestion
                .debug(
                    name: .configuration("Debug"),
                    settings: signingSettings(for: .dev),
                    xcconfig: .relativeToRoot("Config/Debug.xcconfig")
                ),
                .release(
                    name: .configuration("Release"),
                    settings: signingSettings(for: .release),
                    xcconfig: .relativeToRoot("Config/Release.xcconfig")
                )
```